Frontend/Gmfctn/src/app/models/request.ts

```diff
@@ -0,0 +1,4 @@
+interface RequestForAchievement {
```

Comment:
all models (either classes or interfaces) must be export. you need to make following changes:
```suggestion
export interface RequestForAchievement {
```